pkg/controller/gcp/compute/network.go

```diff
@@ -181,7 +181,7 @@ func (c *networkExternal) Delete(ctx context.Context, mg resource.Managed) error
 	_, err := c.Networks.Delete(c.projectID, cr.Spec.Name).
 		Context(ctx).
 		Do()
-	if !clients.IsErrorNotFound(err) {
+	if clients.IsErrorNotFound(err) {
```

Comment:
What was the reason for this change?

Reply:
This was a bad merge from bringing the GCP networking types from the main crossplane repo.  I've put this change into its own commit.